_scss/_retreat.scss

```diff
@@ -184,7 +184,7 @@
 
     .gold-sponsor {
         img {
-            width: 240px;
+            width: 304px;
```

Comment:
is it responsive?

Reply:
Missing an `height: auto;` for all sponsors

Reply:
not speaking about height but about the width. It's fixed at 304px, if it works also on mobile it should be fine. If not a media query can be required